In [1]:
import pandas as pd
import matplotlib.pyplot as plt
import statistics

In [2]:
# reading both csv files
data_gen = pd.read_csv('Data/heart.csv')
data_o2 = pd.read_csv('Data/o2Saturation.csv')

In [3]:
# Creating a reference table for data_gen
data_gen.head(5)

age  sex  cp  trtbps  chol  fbs  restecg  thalachh  exng  oldpeak  slp  \
0   63    1   3     145   233    1        0       150     0      2.3    0   
1   37    1   2     130   250    0        1       187     0      3.5    0   
2   41    0   1     130   204    0        0       172     0      1.4    2   
3   56    1   1     120   236    0        1       178     0      0.8    2   
4   57    0   0     120   354    0        1       163     1      0.6    2   

   caa  thall  output  
0    0      1       1  
1    0      2       1  
2    0      2       1  
3    0      2       1  
4    0      2       1

For reference, the acronyms at the table have the following meaning:
cp : Types of Chest pain (1 - typical angina, 2 - atypical angina, 3 - non-anginal pain, 4 - asymptomatic)
trtbps : resting blood pressure (in mm Hg)
chol : cholestoral in mg/dl
fbs : fasting blood sugar > 120 mg/dl (1 = true; 0 = false)
restecg : resting electrocardiographic results (0 - normal, 1 -  having ST-T wave abnormality (T wave inversions and/or ST elevation or depression of > 0.05 mV), 2 - showing probable or definite left ventricular hypertrophy by Estes' criteria)
thalach : maximum heart rate achieved
exng: exercise induced angina (1 = yes; 0 = no)

In [4]:
# creating a reference table for data_o2
data_o2.head(5)

98.6
0  98.6
1  98.6
2  98.6
3  98.1
4  97.5